style.css

```diff
@@ -129,6 +129,14 @@ button {
   padding: 0 12px;
 }
 
+.vertical-bar{
+  width: 4px;
+  height: 20px;
+  background: #c0c0c0;
+  box-shadow: inset -1px -1px var(--window-frame), inset 1px 1px var(--button-highlight), inset -1px -1px var(--button-highlight), inset 1px 1px var(--window-frame);
+  border-radius: 5px;
```

Comment:
![image](https://user-images.githubusercontent.com/287268/80383185-4182e200-8871-11ea-94ac-4c7d0b6091ff.png)

Do we want a border radius?

Reply:
Actually, I think you're right - the border-radius can be removed.